compiler/passes/ResolveScope.cpp

```diff
@@ -1124,6 +1127,9 @@ ResolveScope::lookupPublicUnqualAccessSyms(const char* name,
               ResolveScope *scope = ResolveScope::getScopeFor(ms->block);
               if (Symbol *retval = scope->lookupNameLocally(nameToUse)) {
                 symbols.push_back(retval);
+
+                reexportPts[retval] = visStmt;
```

Comment:
I am a bit confused here. A symbol can be multiply-defined because of multiple `import`s that reexports that symbol. In that scenario, are we using the last one that we have found while iterating the `mUseImportList`? Should we report all of them?

Or maybe what I think can happen doesn't happen?

Reply:
It looks like we report the last one.  I can see reporting all of them being useful, but I think it would also be sufficient to indicate in the error message that there may be other re-exports along the way (or to allow the reader to draw their own conclusions)

Reply:
I opened #16317 (Engin had said offline it was okay to punt on the topic)